PasteBinService/RegisterService/src/main/java/org/potrohit/springboot/RegisterService/activity/common/RegisterPasteBinRequest.java

```diff
@@ -0,0 +1,13 @@
+package org.potrohit.springboot.RegisterService.activity.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
```

Comment:
Remove line 8,9. @Data already comes with all args. Instaed add @Builder and try to build objects using it.